extkingpin/pathorcontent.go

```diff
@@ -68,19 +68,21 @@ func RegisterPathOrContent(cmd FlagClause, flagName string, help string, opts ..
 func (p *PathOrContent) Content() ([]byte, error) {
 	fileFlagName := fmt.Sprintf("%s-file", p.flagName)
 
-	if len(*p.path) > 0 && len(*p.content) > 0 {
+	path := p.Path()
+	internalContent := p.internalContent()
```

Comment:
I prefer previous code -it is cleaner and `path` and `content` fields are never nil.